htdocs/compta/facture/card.php

```diff
@@ -1093,7 +1093,7 @@
 				$object->fk_incoterms       = GETPOSTINT('incoterm_id');
 				$object->location_incoterms = GETPOST('location_incoterms', 'alpha');
 				$object->multicurrency_code = GETPOST('multicurrency_code', 'alpha');
-				$object->multicurrency_tx   = GETPOSTINT('originmulticurrency_tx');
```

Comment:
Can you use instead GETPOSTFLOAT ?

Reply:
Ok, i see that some others values are not right due to "integer" conversion too
I will fix that too in this pr